requirements-dev.txt

```diff
@@ -0,0 +1,43 @@
+ansicolors==1.1.8
+boto3==1.5.32
+botocore==1.8.50
+docutils==0.16
+jmespath==0.9.3
+python-dateutil==2.8.1
+PyYAML==4.2b4
+s3transfer==0.1.13
+six==1.15.0
+elasticsearch[es6]==6.1.1
+elasticsearch_dsl[es6]==6.1.0
+elasticsearch[es1]==1.9.0
+elasticsearch_dsl[es1]==0.0.11
+nose==1.3.7
```

Comment:
I think all this file needs that requirements.txt doesn't have, is just `nose` and `coverage`, maybe a few others.  I'm guessing this file was autogenerated somehow.

Reply:
Oh. Good point. I kind of combined them together. 

I also included autopep8 and flake8, which were mentioned in the setup.py file.

Reply:
The new requirements-dev.txt file was generated via the following:

```bash
pip3 install nose coverage autoflake autopep8 pylint
pip3 freeze > requirements-dev.txt
```

The packages required by Cloudtracker itself are no longer included in the requirements-dev.txt file and remain in requirements.txt